recipe/meta.yaml

```diff
@@ -11,7 +11,6 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  skip: true  # [win]
```

Comment:
As I do not depend on Python C API at all (I use ctypes and load a shared library), is it possible to use VC 14 build with Python 2? 

Or are msvcrt14 and msvcrt9 not compatible in the same process? Then I believe I could statically link to msvcrt.

Or maybe it is just not worth the pain, and I'll see what to do when Py2 on Windows users will appear.